src/js/views/patients/schedule/schedule_views.js

```diff
@@ -102,12 +102,17 @@ const DayItemView = View.extend({
       {{/if}}
     </td>
     <td class="schedule-list__action-list-cell schedule-list__patient">
-      <div class="schedule-list__state-patient">
-        <span class="schedule-list__action-state action--{{ stateOptions.color }}">{{fa stateOptions.iconType stateOptions.icon}}</span><span class="schedule-list__search-helper">{{ state }}</span>&#8203;{{~ remove_whitespace ~}}
+      <div class="schedule-list__patient-details">
+        <span class="schedule-list__patient-sidebar-icon">
+          <button class="js-patient-sidebar-button">
+            {{far "address-card"}}
+          </button>
+        </span>&#8203;{{~ remove_whitespace ~}}
         <span class="schedule-list__patient-name {{#if isReduced}}is-reduced{{else}}js-patient{{/if}}">{{ patient.first_name }} {{ patient.last_name }}</span>&#8203;
       </div>
     </td>
     <td class="schedule-list__action-list-cell">
+      <span class="schedule-list__action-state action--{{ stateOptions.color }}">{{fa stateOptions.iconType stateOptions.icon}}</span><span class="schedule-list__search-helper">{{ state }}</span>&#8203;{{~ remove_whitespace ~}}
       <span class="schedule-list__action-name {{#unless isReduced}}js-action{{/unless}}">{{ name }}</span>&#8203;{{~ remove_whitespace ~}}
```

Comment:
can `schedule-list__action-name` be removed now?

Reply:
Indeed it can, nice catch 😄